test/mod/test_fmt_v2.py

```diff
@@ -72,8 +72,41 @@
     ]
 }
 
+BAD_SHA = "sha256:d540ca8c5e21ba5f063286c94a088af2aac0b15bc40df6fd562d40154c10f4a1"
+
+BAD_REF_PIPELINE = {
+  "version": "2",
+  "sources": {
+    "org.osbuild.curl": {
+      "items": {
+        "sha256:c540ca8c5e21ba5f063286c94a088af2aac0b15bc40df6fd562d40154c10f4a1": "",
+      }
+    }
+  },
+  "pipelines": [
+    {
+      "name": "build",
+      "stages": [
+        {
+          "type": "org.osbuild.rpm",
+          "inputs": {
+            "packages": {
+              "type": "org.osbuild.files",
+              "origin": "org.osbuild.source",
+              "references": {
+                BAD_SHA: {},
+                "sha256:15a654d32efaa75b5df3e2481939d0393fe1746696cc858ca094ccf8b76073cd": {}
+              }
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
 
-class TestFormatV1(unittest.TestCase):
+
+class TestFormatV2(unittest.TestCase):
```

Comment:
Ups! Good catch! But maybe do it in a separate commit ;)

Reply:
that's how I would prefer it too